### DISCUSSION TITLE
Failed to connect to codespace, error 500

### DISCUSSION THREAD

**Discussion by AsafFisher** [community/community#35969] [Codespaces]
![image](https://user-images.githubusercontent.com/6796683/195467314-d0cc079d-7edb-4610-90e8-e859194c1a58.png)
 What do I do?

**Comment by limyuquan** [comment]
also have the same problem today

**Reply by Isabella-209** [reply]
I have the same problem

**Reply by limyuquan** [reply]
all is working now https://www.githubstatus.com/incidents/7hfyz8mjg663
